src/modules/charge-categories/routes.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const controller = require('./controller');
+const Joi = require('joi');
+module.exports = {
+
+  getChargeCategoryByProperties: {
+    method: 'GET',
+    path: '/water/1.0/charge-categories',
+    handler: controller.getChargeCategoryByProperties,
+    config: {
+      validate: {
+        query: Joi.object().keys({
+          source: Joi.string().required().valid('tidal', 'non-tidal'),
+          loss: Joi.string().required().valid('low', 'medium', 'high'),
+          waterAvailability: Joi.string().required().valid('restricted availablity or no availability', 'available'),
```

Comment:
Is there any reason why `waterAvailability` isn't simply a boolean?

Reply:
changed to isRestrictedSource and aligned with UI